content/docs/schema.md

```diff
@@ -60,7 +60,7 @@ This is my main post body.
 
 Once we've defined a collection, we can edit its fields through the Tina UI, or [query its content](/docs/graphql/overview/) using the Tina Content API.
 
-<iframe width="100%" height="450px" src="https://tina-gql-playground.vercel.app/iframe/string-body" />
+<iframe width="100%" height="450px" src="https://tina-gql-playground.vercel.app/string-body" />
```

Comment:
Just to confirm, did you mean for this one to remain an `iframe` or switch to an `a`?

Reply:
oops good catch, just switched that back to use the iframe url

Reply:
But yeah, i was thinking having one iframe per page is fine, and gives a "taste" of the examples

Reply:
I like that.  👍